_1327/main/models.py

```diff
@@ -83,6 +78,13 @@ def can_edit(self, user):
 	def can_delete(self, user):
 		return self.can_edit(user)
 
+	@classmethod
+	def used_permissions(cls):
+		return (
+		("{app}.{codename}".format(app=ContentType.objects.get_for_model(cls).app_label, codename=cls.VIEW_PERMISSION_NAME), _('view')),
```

Comment:
and how about returning a dictionary instead of a tuple? You could also think of a `namedtuple`. This would make the code for the form handling easier to understand